### PR TITLE
GH-27 Integrate systemd unit files into the supervisor-status folder

### LINKED ISSUE #27: Integrate systemd unit files into the supervisor-status folder
Integrate systemd unit files into the supervisor-status folder.

### PR DESCRIPTION
**Pull request by johnsonw**
Fixes #27.

Integrate systemd unit files into the supervisor-status folder.

Signed-off-by: Will Johnson <william.c.johnson@intel.com>

### CODE REVIEW

File: supervisor-status/supervisor-status.spec [hunk 1]
```diff
@@ -1,13 +1,15 @@
 %{?nodejs_find_provides_and_requires}
 
-Name:       supervisor-status
-Version:    1.0.0
+Name:       iml-supervisor-status
+Version:    1.0.1
 Release:    2%{?dist}
```
Comment (brianjmurrell) [inline supervisor-status/supervisor-status.spec]: Now that a new version is being packaged, the `Release:` should be reset back to 1.

File: supervisor-status/supervisor-status.spec [hunk 2]
```diff
@@ -29,33 +31,33 @@ It is meant to be used with unix domain sockets + socket activation.
...
+systemctl stop %{name}.service
+systemctl disable %{name}.service
+systemctl stop %{name}.socket
+systemctl disable %{name}.socket
+rm /var/run/%{name}.sock
 
 %changelog
-* Wed Jun 14 2017 Joe Grund
-- new package built with tito
+* Wed Jul 27 2017 Will Johnson <william.c.johnson@intel.com> - 1.0.1-2
```
Comment (brianjmurrell) [inline supervisor-status/supervisor-status.spec]: You should put in here what has changed in the spec.

Also update the release per the previous comment.

File: supervisor-status/supervisor-status.spec (hunk 1 above)
Comment (johnsonw) [inline supervisor-status/supervisor-status.spec]: Ok, do I need to re-run tito after making this change?

Comment (jgrund) [inline supervisor-status/supervisor-status.spec]: Just delete the tito commit and force push. Then after making the change, run tito again.

File: supervisor-status/supervisor-status.spec [hunk 3]
```diff
@@ -29,33 +31,35 @@ It is meant to be used with unix domain sockets + socket activation.
...
 %files
-%attr(0744,root,root)/usr/sbin/supervisor-status
-%attr(0744,root,root)/usr/lib/systemd/system/supervisor-status.service
-%attr(0744,root,root)/usr/lib/systemd/system/supervisor-status.socket
+%attr(0755,root,root)/usr/lib/%{name}/supervisor-status
+%attr(0644,root,root)/usr/lib/systemd/system/%{name}.service
+%attr(0644,root,root)/usr/lib/systemd/system/%{name}.socket
 
 %post
-systemctl enable supervisor-status.socket
```
Comment (jgrund) [inline supervisor-status/supervisor-status.spec]: Can we make sure we scope this and the preun block?

**Pull request merged**